draft-tschofenig-rats-psa-token.md

```diff
@@ -531,30 +532,33 @@ keys.
 # Verification
 
 To verify the token, the primary need is to check correct formation and signing
-as for any CWT token.  In addition though, the verifier can operate a policy
-where values of some of the claims in this profile can be compared to reference
-values, registered with the verifier for a given deployment, in order to
-confirm that the device is endorsed by the manufacturer supply chain.  The
-policy may require that the relevant claims must have a match to a registered
-reference value.  All claims may be worthy of additional appraisal.  It is
-likely that most deployments would include a policy with appraisal for the
-following claims:
-
-* Instance ID - the value of the Instance ID can be used (together with the kid
-  in the token COSE header, if present) to assist in locating the public key
-  used to verify the token signature.
+as detailed in {{sec-token-encoding-and-signing}}.  In particular, the Instance
+ID claim is used (together with the kid in the token COSE header, if present)
+to assist in locating the public key used to verify the token signature.
+
+In addition, the Verifier will typically operate a policy where values of some
+of the claims in this profile can be compared to reference values, registered
+with the Verifier for a given deployment, in order to confirm that the device
+is endorsed by the manufacturer supply chain.  The policy may require that the
+relevant claims must have a match to a registered reference value.  All claims
+may be worthy of additional appraisal.  It is likely that most deployments
+would include a policy with appraisal for the following claims:
+
 * Implementation ID - the value of the Implementation ID can be used to
   identify the verification requirements of the deployment.
 * Software Component, Measurement Value - this value can uniquely identify a
-  firmware release from the supply chain. In some cases, a verifier may
+  firmware release from the supply chain. In some cases, a Verifier may
   maintain a record for a series of firmware releases, being patches to an
   original baseline release. A verification policy may then allow this value to
   match any point on that release sequence or expect some minimum level of
   maturity related to the sequence.
 * Software Component, Signer ID - where present in a deployment, this could
-  allow a verifier to operate a more general policy than that for Measurement
+  allow a Verifier to operate a more general policy than that for Measurement
   Value as above, by allowing a token to contain any firmware entries signed by
   a known Signer ID, without checking for a uniquely registered version.
+* Certification Reference - if present, this value could be used as a hint to
+  locate an entry in the PSA Certification database associated with the
```

Comment:
There may or may not be a single database or certification scheme. To address this, might be better to rephrase "locate security certification information associated with the attesting device. An example could be a certificate number issued by www.psacertified.org" 

What do you think?

Reply:
thanks!  Your suggested text is in b45adbe